Makefile

```diff
@@ -1,42 +1,62 @@
 SHELL=/usr/bin/env bash -o pipefail
 
 IMAGE = monitoring-stack-operator
-VERSION?=$(shell cat VERSION)
+VERSION ?= $(shell cat VERSION)
 
-TOOLS_BIN_DIR=$(shell pwd)/tmp/bin
-export PATH := $(TOOLS_BIN_DIR):$(PATH)
-export GOBIN := $(TOOLS_BIN_DIR)
+# running make builds the operator (default target)
+all: operator
 
-TOOLS=\
-	$(TOOLS_BIN_DIR)/controller-gen \
-	$(TOOLS_BIN_DIR)/golangci-lint
 
-$(TOOLS_BIN_DIR):
-	mkdir -p $(TOOLS_BIN_DIR)
+## Tools
 
-$(TOOLS): $(TOOLS_BIN_DIR)
-	go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.1
+TOOLS_DIR = $(shell pwd)/tmp/bin
+CONTROLLER_GEN=$(TOOLS_DIR)/controller-gen
+GOLANGCI_LINT=$(TOOLS_DIR)/golangci-lint
+
+$(TOOLS_DIR):
+	@mkdir -p $(TOOLS_DIR)
+
+$(CONTROLLER_GEN): $(TOOLS_DIR)
+	@{ \
```

Comment:
What does this syntax do?

Reply:
executes all commands in a single shell "process" so that
```
{
  FOO=bar
  echo $FOO
}
```
prints `bar` instead of "" if it wasn't in the same "shell"